test/plugin/secretless_test_harness/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+	goPlugins "plugin"
+
+	secretlessLog "github.com/cyberark/secretless-broker/internal/log"
+	tcpinternal "github.com/cyberark/secretless-broker/internal/proxyservice/tcp"
+	"github.com/cyberark/secretless-broker/pkg/secretless/plugin/connector/tcp"
+)
+
+// ListenPort is the port that the echo server is listening on
+const ListenPort = "6175"
```

Comment:
This should be probably randomized/passed-in to avoid parallel-build conflicts

Reply:
It's intended for manual testing so the port definitely needs to be predictable.

Reply:
What about having a default but capability to be over-ridden by CLI args for automation?

Reply:
Hmm.. want to ping me on slack or was this resolved on accident?